src/blog/2023/004-how-jmix-uses-bpmn-js.md

```diff
@@ -0,0 +1,186 @@
+- You can upload BPMN diagrams prepared by an analyst directly into IntelliJ IDEA and then enrich them with project-specific business logic, user screens, integrations, and exception handling without the need to switch to one another tool;
+- You can deploy and run it on the developer’s workstation and test its business logic to ensure it runs as intended. Here you get access to all of IntelliJ's powerful debugging tools to streamline the process and make it more efficient;
+- When finished with local debugging, you can commit and push the changes into the shared repository, allowing the analyst to see the updated version of the BPMN diagram and check the live version of the diagram within the application running in the test environment;
+- If the analyst needs to change the BPMN model, the diagram can be adjusted in the application runtime using the same tool and tested again. After successful testing, the updated diagram version can be downloaded and sent back to the developer for implementation of the changes.
+
+Thus, with [Jmix](http://jmix.io/?utm_source=bpmn&utm_medium=referral&utm_campaign=bpnmio-article), teams can cover the entire business process development lifecycle within the known tools, and what’s more, it can be integrated within CI/CD pipelines. In real projects, developers add dashboards to implement various process analytics and acquire statistics. This approach is quite similar to a BPMS solution that is tailored to a data-centric application and business logic in Java or Kotlin. 
+
+Thanks to the [bpmn.io project](http://bpmn.io/), developers can address diverse use cases with the same easy-to-use and widely applicable tooling, in a standalone designer, as part of their IntelliJ IDEA or embedded inside their applications.
+
+---
+
+## About the Author
+
+[Maxim Gorbunkov](https://github.com/gorbunkov) is a Jmix developer and works at [Haulmont](https://haulmont.tech/?utm_source=bpmn&utm_medium=referral&utm_campaign=bpnmio-article), where we’re focused on developing tools and platforms that enhance developers’ productivity by automating coding routines and bringing ready-to-use application components into a single development environment — IntelliJ IDEA. Established in 2008 as a custom development software house, we understand what productivity means for technical leaders. When time and resources are limited, selecting the right tools for the job and capitalising on framework strengths rather than fighting against them is essential. This led to the creation of a high-productivity platform for Java enterprise applications. In 2016 our team introduced the CUBA Platform as open-source, and in 2020, we released the CUBA Platform successor — [Jmix](http://jmix.io/?utm_source=bpmn&utm_medium=referral&utm_campaign=bpnmio-article).
```

Comment:
@nikku The author is Viktor Fadeev, not me. He just asked me to publish the PR:)

Reply:
And how shall I describe him? :)

Reply:
Viktor's position: Jmix Product Marketing Manager
LinkedIn profile: https://www.linkedin.com/in/viktor-fadeev-platform

Reply:
I've updated the author info in the article. We'll do a final review and I will come back to you.